enhancements/upgrade-readiness.md

```diff
@@ -97,13 +97,15 @@ There are many reasons why operators should communicate their conditions through
 - By writing the condition to the status of an owned CustomResource, it becomes very clear which CR is responsible for changing OLM behavior.
 - Introducing an OLM owned CRD forces operators to integrate with OLM at a much deeper level than writing to a resource that they already manage. Operators would be required to detect if it is being managed by OLM and begin writing to the API. By writing to an owned CR, the Operator does not require OLM specific code and can be run independently of OLM.
 
-Allowing operators to write to the status of any CR creates a potential scaling issue if enough CRs are on cluster. OLM will avoid these performance issues by introducing the Status Probe Operator (SPO) which will reconcile the `probe` resource. If an operator chooses to write to their own CRs, SPO will create a pod that will keep the status of the `probe` resource synced with the conditions in the CRs.
+Allowing operators to write to the status of any CR creates a potential scaling issue if enough CRs are on cluster. OLM will avoid these performance issues by introducing the Status Probe Operator (SPO) which will reconcile the `Probe` resource. If an operator chooses to write to their own CRs, SPO will create a pod that will keep the status of the `Probe` resource synced with the conditions in the CRs.
+
+>Note: Unfortunately, OLM will not support an operator communicating its status via cluster-scoped CRs. If an operator is scoped to a namespace but writes conditions to a cluster-scoped CR, OLM would be unable to identify which operator is communicating its status if multiple installs of the operator were present on cluster.
 
 ### Opting into OLM Supported Conditions
 
-Operators may opt-in to "OLM Supported Conditions" by adding annotations to their owned CRD(s), where the key is the defined by OLM and the value is a boolean expression over condition types applied to the CR at `.status.conditions`. The "OLM Supported Condition" annotation key will adhere to the following format: `olm.Condition.<ConditionName>`.
+Operators may opt-in to "OLM Supported Conditions" by adding annotations to their owned CRD(s), where the key is the defined by OLM and the value is a boolean expression over condition types applied to the CR at `.status.conditions`. The "OLM Supported Condition" annotation key will adhere to the following format: `operators.coreos.com/condition.<ConditionName>`.
```

Comment:
don't we have an operatorframework.io prefix that we use now?

Reply:
Afterlooking through OLM code, it looks like we user `operators.operatorframework.io` elsewhere. We could update this annotation to `operator.operatorframework.io/condition.<ConditionName>`, thoughts?

Reply:
I should mention that there are 40 characters in `operator.operatorframework.io/condition.`, leaving only 23 characters for Condition Type names due to [existing k8s restraints](https://kubernetes.io/docs/concepts/overview/working-with-objects/annotations/#syntax-and-character-set) on  annotation key lengths.

Reply:
`operators.operatorframework.io/status.` saves us a few chars.

Reply:
@ecordell -- The `Operator` API uses `operators.coreos.com`, should we change it to match this? 

nit: I can't help but feel like `operators.operator...` is needless stutter; maybe just `operatorframework.io` is prettier?

Reply:
I support `operatorframework.io` as the prefix - might be difficult to change it in all places though.